content/handbook/sales/process/version-check/index.md

```
@@ -0,0 +1,133 @@
+
+レスポンスが `null` の場合、いくつかのステップを取ることができます:
+
+- ネットワークが `version.gitlab.com` への外部 API リクエストを許可／ホワイトリスト登録していることを確認します。
+- Sidekiq でバックグラウンドジョブがスケジュールされていて、何かの後ろで [スタックしていない](#ensure-sidekiq-jobs-are-in-healthy-state) ことを確認します。
+
+### Sidekiq ジョブが健全な状態にあることを確認する
+
+`/admin/background_jobs` にアクセスすることで、インスタンスでスケジュール／実行／保留中のジョブを調べることができます。
+探すべきジョブは `external_service_reactive_caching` と呼ばれています。
+
+[内部 API](#use-the-internal-api-to-check-the-cache) が `null` を返している場合、またはアップグレードバッジが欠けている場合:
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)